ui/src/organizations/components/TaskRow.tsx

```diff
@@ -0,0 +1,48 @@
+// Libraries
```

Comment:
there is already a `TaskRow` component in src/tasks/components, is there any way we could use that component here instead of making a new one?

Reply:
Going to create a new issue for this